content/blog/qmstr-sprint-4.1-report.md

```diff
@@ -0,0 +1,84 @@
++++
+title = "Quartermaster Sprint 4.1 Development Report: Go, Java and Python integration libraries"
+date = "2019-02-20T08:00:00+01:00"
+tags = ["qmstr", "FOSS", "compliance", "sprint", "report"]
+categories = ["qmstr"]
+banner = "img/blog/qmstr-sprint-4.1-title.jpg"
++++
+
+The key goal of QMSTR milestone 4 is to stabilize the APIs and
+streamline the documentation and tutorials to prepare the integration
+with the development work that takes part in
+the [FASTEN project](https://www.fasten-project.eu/) and
+in
+[ACT](https://www.linuxfoundation.org/press-release/2018/12/the-linux-foundation-to-launch-new-tooling-project-to-improve-open-source-compliance/). To
+prepare for that, sprint 4.1 focused on refactoring the integration
+libraries for Go, Java and Python that developers use to
+create [QMSTR](https://github.com/QMSTR) modules and client side
+tools. These libraries help users to create diverse sets of modules
+and client side code to fit their needs, and facilitate the
+integration with various services like CI/CD environments.
+
+<!--more-->
+
+# Quartermaster Sprint 4.1
+
+A QMSTR sprint spans two weeks. Starting with milestone 4, sprints
+begin on Wednesdays and end on Tuesdays. The development work is
+hosted at the QMSTR Github organization. You can follow the progress
+of the ongoing sprint for example by looking at
+the [project board](https://github.com/orgs/QMSTR/projects/1).
+
+## About modules and clients for QMSTR
+
+Users can integrate with QMSTR either by writing modules or from
+client side code. Modules are usually analyzers or reporters. They are
+individual programs that are running in the context of the master and
+communicate with the master locally. Client side software can either
+be command line tools that are integrated for example with build
+systems, or code running in services like CI systems.
+
+Both modules and clients talk to the master
+using [gRPC](https://grpc.io/) and can therefore be implemented in
+languages with gRPC support. The libraries implement common
+functionality that make it easier for implementers and focus on Java,
+Go and Python.
+
+Both for module and client side programmers it is important to
+understand the licensing perspective of QMSTR: Modules and clients are
+separate binaries, and the QMSTR code these binaries use comes under
```

Comment:
"and the QMSTR code these binaries use comes under the terms of the LGPL"
I think in this part of the sentence there is a word missing.